test/package.spec.ts

```diff
@@ -639,10 +640,11 @@ describe('test', async () => {
         .post('/api/groups/search')
         .send(body)
         .set('Accept', 'application/json')
-        .set('Cookie', [`sessionToken=${sessionToken}`])
```

Comment:
Why you deleted cookie?

Reply:
Because when I'm running group tests, I'm running only the group tests to not waste time retesting the whole standard flow

Reply:
you can move let sessionToken on level up and use this